kotlin-emotion/src/main/kotlin/emotion/css/keyframes.kt

```diff
@@ -0,0 +1,12 @@
+@file:JsModule("@emotion/css")
+@file:JsNonModule
+
+package emotion.css
+
+import csstype.AnimationName
+import csstype.Percentage
+import csstype.Properties
+import kotlinx.js.Record
+
+@JsName("keyframes")
+external fun rawKeyframes(blocks: Record<Percentage, Properties>): AnimationName
```

Comment:
Could we use original name?

Reply:
do you propose overload?

Reply:
I think original signature would be very unuseful for end user, DSL overload is more similar to original DSL from emotion

Reply:
> do you propose overload?
Yes

Reply:
but what about my point? Do you think it is not a problem?

Reply:
after discussion it was decided to keep original signature public because it does not conflict with extension